docs/proposals.md

```diff
@@ -0,0 +1,14 @@
+# Proposals
```

Comment:
is this related to removing MostRequestedPriority?

Reply:
no, it is related to https://github.com/kubernetes-sigs/descheduler/issues/646, i just thinking that for use one pr for fix two document related issues. is it change the pr title will be better?

Reply:
technically if it's unrelated it should probably be a different PR, to organize them better. Hope you don't mind

Reply:
get it, i don't mind :) i will put this on a different pr :)